refactor(http): remove https from server

jniles opened the pull request:
**BREAKING CHANGE**

This commit removes HTTPS support from bhi.ma.  HTTPS is best handled by a reverse proxy.  See [this article](https://expressjs.com/en/guide/behind-proxies.html) for more details on running the server behind a proxy.

Because of this change, there is no more need to store self-signed certificates.  They have been removed.

To update your code you should:
1. Remove TLS_\* variables from your .env.\* files.
2. Remove HTTPS=1 from your .env.\* files.
3. Clear the browser cache.

Closes #455.

---

Thank you for contributing!

Before submitting this pull request, please verify that you have:
- [x] Run your code through JSHint.  [Check out our styleguide](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/docs/STYLEGUIDE.md).
- [x] Run integration tests.
- [x] Run end-to-end tests.
- [x] Accurately described the changes your are making in this pull request.

For a more detailed checklist, [see the official review checklist](https://docs.google.com/document/d/1nupLVLRXgSZJQo_acLgrwvPnN8RukfSiwRhSToj81uU/pub) that this PR will be evaluated against.

Ensuring that the above checkboxes are completed will help speed the review process and help build a stronger application.  Thanks!

Commit:
refactor(http): remove https from server

**BREAKING CHANGE**

This commit removes HTTPS support from bhi.ma.  HTTPS is best handled by
a reverse proxy.  See [this
article](https://expressjs.com/en/guide/behind-proxies.html) for more
details on running the server behind a proxy.

Because of this change, there is no more need to store self-signed
certificates.  They have been removed.

To update your code you should:
 1. Remove TLS_* variables from your .env.* files.
 2. Remove HTTPS=1 from your .env.* files.
 3. Clear the browser cache.

Closes #455.

DedrickEnc commented on server/config/express.js:
```diff
@@ -43,10 +43,16 @@ exports.configure = function configure(app) {
     retries: 50
   }));
 
+  // indicate that we are running behind a trust proxy and should use a secure cookie
```
What will happen if we are in a development environnement

jniles replied:
Nothing.  The application will not do any extra checks on the cookie for tampering or check HTTP headers.

DedrickEnc commented on server/config/express.js:
```diff
@@ -43,10 +43,16 @@ exports.configure = function configure(app) {
     retries: 50
   }));
 
+  // indicate that we are running behind a trust proxy and should use a secure cookie
```
According to the article the sub net or some address should be set even if it is not important so what will be our position we will configure the sub net or not? if yes how? (Notice that we will face many type of network in terms of address configuration)

jniles replied:
There address is local.  There is no configuration necessary, the only thing we need to do is set `trust proxy` to `true` in production and listen on a high number port (like 3000).

We can then put any proxy (Apache, Nginx, HAproxy, etc) on port 80 and 443 of the same machine and route all traffic to localhost:3000 (where the express server is).

For a boolean value, nothing else needs to be configured.

DedrickEnc reviewed (commented):
@jniles 
I expressed some concerns

mbayopanda reviewed (approved):
The migration from https to http is so clear 👍 and it works very well.

Commit:
fix(test): end-to-end tests run full suite

This commit removes the `describe.only()` tag from the user end to en
test so that all tests run.
